Resources/Prototypes/Entities/Objects/Devices/pda.yml

```
@@ -874,6 +1040,14 @@
     uiKey: enum.PdaUiKey.Key
     preinstalled:
       - NotekeeperCartridge
+    cartridgeSlot:
+      priority: -1
+      name: Cartridge
+      ejectSound: /Audio/Machines/id_swipe.ogg
+      insertSound: /Audio/Machines/id_insert.ogg
+      whitelist:
+        components:
+          - Cartridge
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)